test/TestIncludeMissFileWithRole.py

```diff
@@ -93,7 +93,7 @@ def test_cases_warning_message(runner, caplog):
         if "Couldn't open" in str(record):
             noexist_message_count += 1
 
-    assert noexist_message_count == 1
+    assert noexist_message_count == 3  # 3 retries
```

Comment:
Why would it be exactly 3?

Reply:
Because this is what we added as maximum retries. That test is testing that an invalid path does trigger the warning. It fails and needs to fail with 3 warnings not one like previously.